src/components/LeftSidebar/LeftSidebar.spec.js

```diff
@@ -26,6 +26,13 @@ jest.mock('../../services/conversationsService', () => ({
 // short-circuit debounce
 jest.mock('debounce', () => jest.fn().mockImplementation(fn => fn))
 
+window.navigator.locks = {
+	request: jest.fn().mockImplementation((name, callback) => {
+		callback()
+		return new Promise(() => {})
+	}),
```

Comment:
`new Promise(() => {})` is an infinity promise... It should be `return Promise.resolve()`. And I have no idea why it works 😅